gbus/bus.go

```diff
@@ -281,6 +283,23 @@ func (b *DefaultBus) NotifyHealth(health chan error) {
 	b.healthChan = health
 }
 
+//GetHealth implements Health.GetHealth
+func (b *DefaultBus) GetHealth() HealthCard {
+	dbInitiated := b.TxProvider != nil
+	var dbConnected bool
+
+	if dbConnected = false; dbInitiated {
+		dbConnected = b.TxProvider.Ping(b.DbPingTimeout)
+	}
+
+	return HealthCard{
+		DbInitiated:        dbInitiated,
```

Comment:
Not sure this should be on the HealthCard what does DbInitiated mean ?

Reply:
from outside I want to know if to check the DbConnected value in the health check. If the grabbit is not transactional and only uses rabbit the health check should reflect that.

Reply:
Is `UsingDb`  or `IsTransactional` better?

Reply:
once the bus is configured to be transactional then it uses a DB and the health check should include checking the DB. so it is enough for a caller to configure the bus as transactional and for the health check to check whether IsTxnl